### DISCUSSION TITLE
⏳ 🔗 Multi Color Gradient Background

### DISCUSSION THREAD

**Discussion by heroicP** [elementor/elementor#15512] [Feature Request]
I know I can do this with some custom CSS but It would be nice if the gradient background interface allowed the user to set multiple colors for the gradient instead of just two.

**Comment by asadkn** [comment]
Really important for consistency. Users are getting used to multi-color gradients as they're native to the post editor. 

For inspiration, see GradientPicker in native @wordpress/components.

**Comment by fabigrafo** [comment]
Jup, +1
